modules/mobile/spec/request/claims_and_appeals_overview_request_spec.rb

```diff
@@ -3,16 +3,27 @@
 require 'rails_helper'
 require_relative '../support/iam_session_helper'
 require_relative '../support/matchers/json_schema_matcher'
+require 'lighthouse/benefits_claims/configuration'
+require 'lighthouse/benefits_claims/service'
 
-RSpec.describe 'claims and appeals overview', type: :request do
-  include JsonSchemaMatchers
+RSpec.shared_examples 'claims and appeals overview' do |lighthouse_flag|
```

Comment:
yeah it was helpful in spec development to see what the actual differences were. there were a lot of value differences since the test data were totally different claims.